routing/routing_session.cpp

```diff
@@ -339,6 +341,55 @@ SessionState RoutingSession::OnLocationPositionChanged(GpsInfo const & info)
   return m_state;
 }
 
+string FormatRoadDestination(string const & v)
+{
+  // Normalization. "a ;b;  c,d "  " -> "a; b; c; d".
+  string r;
+  strings::Tokenize(v, " ;,", [&](std::string_view d)
```

Comment:
So "New York" destination will be replaced with "New; York" :)

Reply:
Better to leave ";," separators and call strings::Trim(d) inside.

Reply:
OK